moto/batch/models.py

```diff
@@ -1045,34 +1035,30 @@ def create_compute_environment(
 
         if self.get_compute_environment_by_name(compute_environment_name) is not None:
             raise InvalidParameterValueException(
-                "A compute environment already exists with the name {0}".format(
-                    compute_environment_name
-                )
+                f"A compute environment already exists with the name {compute_environment_name}"
             )
 
         # Look for IAM role
         try:
             self.iam_backend.get_role_by_arn(service_role)
         except IAMNotFoundException:
             raise InvalidParameterValueException(
-                "Could not find IAM role {0}".format(service_role)
+                f"Could not find IAM role {service_role}"
             )
 
         if _type not in ("MANAGED", "UNMANAGED"):
             raise InvalidParameterValueException(
-                "type {0} must be one of MANAGED | UNMANAGED".format(service_role)
+                f"type {_type} must be one of MANAGED | UNMANAGED"
```

Comment:
Bugfix - having 'service_role' in the error message was probably a copy/paste error